src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,225 @@
+/* stylelint-disable no-descending-specificity */
+
+.section-title {
+    display: flex;
+    margin: 30px 0 10px 16px;
+}
+
+.section-title__title {
+    max-width: 75%;
+
+    > h2 {
+        margin: 0;
+    }
+}
+
+.section-title__subtitle {
+    color: @color-grey5;
+}
+
+.section-title__cta {
+    margin: 0 16px;
+    white-space: nowrap;
+
+    > a {
+        border-left: 2px solid @color-grey2;
+        display: inline-block;
+        padding-left: 16px;
+    }
+}
+
+.section-title__cta--no-text {
+    margin: 0;
+
+    > a {
+        border: 0;
+        padding: 0;
+    }
+
+    > a > span:not(.section-title__cta-icon) {
+        display: none;
```

Comment:
How come we have a `display: none;` on all spans except the icon.
If we are trying to hide a no-text, someone could add text without being inside a span.
Trying to understand the reasoning behind this.

Also since we already have the anchor tag nested, we can add this section inside the previous `> a` and save some keystrokes.  (although not super critical)

Reply:
@agliga For this specific line, it's not needed. I removed it.

I use similar styling under small-screen media query (which I will refactor later). Reasoning is we don't want to show "See All" text next to arrow icon because small screens have limited space. As for hiding the text inside `a`, is there a way I can hide the next and not the arrow icon? Text and arrow icon are both inside the `a`.

Reply:
Would `.clipped` work?

Reply:
@seangates I tried adding `.clipped()` mixin, but it also makes the `->` icon disappear, which we still want to show.

@agliga @seangates Another alternative is we don't handle this in CSS, but in our documentation we call out "See All" text should not be populated for small screens.

Otherwise, I think we'll need to keep `> a > span:not(.section-title__cta-icon) {` selector for small screens. In eBayUI, we would put the "See All" text inside a span, so I don't have concerns there. For users only using Skin, if they don't use span then we would still show the text.

Reply:
Got it, I think that's a valid reason. I do think it's good to hide stuff that's not supposed to be shown in mweb.
I do think we should also add a container class and hide it that way. Like a `section-title__see-all` or something, and then hide that specifically. That way it doesn't have this weird selector and we can just do `> a > .section-title__see-all` or whatever we name it.

Reply:
I added `section-title__cta-text` class to span with text and the style selector is now just `.section-title__cta-text { display: none }` and `.section-title__cta-text { display: inline }`, depending on if it should be shown or not.